src/utils/cors.rs

```diff
@@ -30,9 +29,10 @@ impl Fairing for CORS {
             response.set_header(Header::new("Access-Control-Allow-Credentials", "true"));
         }
 
-        if request.method() == Method::Options {
+        if request.method() == Method::Options && request.route().is_none() {
```

Comment:
It might make sense to put all of this into 1 `if` (and maybe remove the `response.content_type() == Some(ContentType::JSON)` condition) ... but would have to test this :thinking: